src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -19,16 +21,18 @@ interface ValidatedTextInputPropTypes extends TextInputProps {
   placeholder?: string;
 }
 
-interface HookValidatedTextInputPropTypes extends TextInputProps {
-  dataTestId?: string | undefined;
-  ouiaId?: string;
-  ariaLabel: string | undefined;
-  value: string;
-  placeholder?: string;
-  stepValidation: StepValidation;
-  fieldName: string;
-  warning?: string;
-}
+type HookValidatedTextInputPropTypes = TextInputProps &
+  TextAreaProps & {
+    dataTestId?: string | undefined;
+    ouiaId?: string;
+    ariaLabel: string | undefined;
+    value: string;
+    placeholder?: string;
+    stepValidation: StepValidation;
+    fieldName: string;
+    warning?: string;
+    typeInput?: string;
```

Comment:
Instead of making the type for this 'string', let's make the API more user friendly and explicit and set the type to either `'textInput' | 'textArea'`.

Then we could also rename this to 'type', which would be a bit more idiomatic for Patternfly-esque components.

Reply:
we already have type, so I am changing it to inputType